lib/server.js

```diff
@@ -238,7 +243,7 @@ function handleResponse(opts, req, resp, response) {
         }
     } else {
         opts.log('error/request', {
-            req: req,
+            root_req: req,
```

Comment:
@gwicke yep, I thing the same problem existed before. I'm OK with a follow up, just let's not forget about it